ai_economist/foundation/components/simple_revenue.py

```diff
@@ -0,0 +1,667 @@
+from copy import deepcopy
+
+import numpy as np
+
+from ai_economist.foundation.base.base_component import (
+    BaseComponent,
+    component_registry,
+)
+from ai_economist.foundation.components.utils import (
+    annealed_tax_limit,
+    annealed_tax_mask,
+)
+
+
+@component_registry.add
+class SimpleRevenue(BaseComponent):
```

Comment:
Hi @sarahlc888  -- SimpleRevenue seems to still be a copy of the old tax component? Are you still working on this, or is this no longer needed?

Reply:
It is the same as the existing component but adds tax revenue to the planner coin endowment rather than redistributing it. (See line 389 for example.) I realize that the code is very redundant—is there an alternative approach you had in mind? I was not sure how to go about this.